src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -123,49 +123,16 @@ protected override void ProcessRecord()
 
                     break;
             }
-
+            
```

Comment:
Please revert. Below too.
Please turn on removing tail spaces in your editor.

Reply:
Please revert which parts, and rever them to what state?
I will check for tail spaces.

Reply:
My request is only for tail spaces. Editor can automatically do this for you.

Reply:
OK.
I'm using the GitHub web editor, it can't automatically remove tail spaces. I was only making a quick edit to add `using` and was not setup for a larger refactor. I think I have removed the tail spaces.